src/utils/growthbook.js

```diff
@@ -0,0 +1,6 @@
+/* eslint-disable import/prefer-default-export */
+
+import CreateUserExperimentMutation from './mutations/CreateUserExperimentMutation'
+
+export const trackingCallback = (experiment, result) =>
+  CreateUserExperimentMutation(result.hashValue, experiment.key, result.value)
```

Comment:
Is hashValue going to be the user ID or the hashed output of the user ID input? I couldn't find detailed info in the docs.